### DISCUSSION TITLE
Iron Golem It will form on the fence and the lower half of the brick.

### DISCUSSION THREAD

**Discussion by jennings365** [PurpurMC/Purpur#1006] [Feature Requests]
Iron Golem It will form on the fence and the lower half of the brick.
Please look at pictures
(https://user-images.githubusercontent.com/107594608/174200805-47774d01-eb37-42ed-a546-3e94b87ed40d.png)

I don't remember any configuration files modifying him, but he showed up.

**Comment by BillyGalbreath** [comment]
Mojang modified the iron golem spawning code quite a bit. Can you check if that happens on vanilla, too?
